pandas/tests/reductions/test_describe_ndframe.py

```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
+
+"""
+We test the describe_ndframe function.
+"""
+
+import numpy as np
+import pytest
+
+from pandas.core.methods.describe import _refine_percentiles
+
+@pytest.mark.parametrize(
+    "percentiles_, expected", [
+        (None, np.array([0.25, 0.5, 0.75])),
+        ([], np.array([0.5])),
+        ([0.3, 0.6], np.array([0.3, 0.6])),
+    ]
+)
+def test_refine_percentiles(percentiles_, expected):
```

Comment:
Could you add a test in `pandas/tests/frame/methods/test_describe.py` instead? It should also test the `describe` method directly as mentioned in the original issue